### DISCUSSION TITLE
Known Compatibility with Other HF Transformers Models

### DISCUSSION THREAD

**Discussion by gserapio** [outlines-dev/outlines#698] [Q&A]
First, a huge thanks to the folks that make Outlines possible! Sorry for the naïve question, but is there a running list of what Transformers models are known to work well with Outlines? Or guidelines on what's generally required from a Transformers model to be compatible? For instance, I've gotten Mistral, Mixtral, Phi, and GPT-2 to work, but Google's latest Gemma model throws a `KeyError`. Short reprex:

```
import outlines

model = outlines.models.transformers("google/gemma-2b", device="auto")
```

Thanks in advance!

**Comment by lapp0** [comment]
Do the official examples work for you? 

https://huggingface.co/google/gemma-2b#running-the-model-on-a-cpu

Maybe you haven't configured your HF auth, or haven't accepted the terms and conditions?

**Reply by gserapio** [reply]
Stupid question on my part! Just as I posted this, it worked after I forced an update of Transformers
